### ISSUE TITLE
Could not reproduce imagenet-resnet50 accuracy using keras

### ISSUE THREAD

**Issue by YochayTzur** [tensorpack/tensorpack#1489]
Hi Tensorpack team!

I'm trying to reproduce resnet50 validation accuracy (75%) using the code in:
https://github.com/tensorpack/tensorpack/blob/master/examples/keras/imagenet-resnet-keras.py
running on 8 titanx gpus, with TOTAL_BATCH_SIZE=512. 
First experiments use the code as is. Maximal validation accuracy is 71.5%.
Second experiments add weight decay as described [here](https://jricheimer.github.io/keras/2019/02/06/keras-hack-1/#:~:text=Weight%20decay%2C%20or%20L2%20regularization,decrease%20during%20the%20training%20process.) with alpha=1e-5. Still same accuracy.
Is there anything I should change in the code / environment in order to get the 75%?
Thanks,
Yochay

**Comment by ppwwyyxx** [comment]
As https://github.com/tensorpack/tensorpack/tree/master/examples/keras#note says the keras support is experimental and perhaps it break at some point. Perhaps we should just delete it at some point.

Using the code in https://github.com/tensorpack/tensorpack/tree/master/examples/ResNet will get you the right accuracy.

**Comment by YochayTzur** [comment]
Thanks for the quick answer.
[this comment ](https://github.com/tensorpack/tensorpack/tree/master/examples/keras#imagenet-example) states that keras imagenet "eproduce exactly the same" as resnet example. Was this tested?

**Comment by ppwwyyxx** [comment]
It was tested. The comment also says "a complicated model or a future version of Keras may break them."

**Comment by ppwwyyxx** [comment]
The keras example is now removed.
